README.md

```diff
@@ -8,6 +8,28 @@ Clementine is a modern music player and library organizer for Windows, Linux and
 - Buildbot: http://buildbot.clementine-player.org/grid
 - Latest developer builds: http://builds.clementine-player.org/
 
+Opening an issue
+----------------
+### Ask a new feature
+
+Please :
+
+ * Check if the new feature is not already implemented (Changelog)
+ * Check if another person didn't already open an issue
+ * If there is already an opened issue there is no need to comment "+1", it won't help :o)
+
+### Report a bug
+
+Please :
+ 
+ * Try with the latest developer build to see if the bug is still present. If if works like a charm although you see an opened issue please comment it to explain the issue was fixed
```

Comment:
Would be nice to add the link: http://builds.clementine-player.org/
Maybe warn users that those builds aren't stable release, and might not work well, and sometimes break things like user settings?
Unfortunately, we currently don't have "profiles" (like Firefox for examples) so users can try developers builds with a "Test" profile to be sure they will not break anything in their current profile.